mp_locales/pl_PL.json

```
@@ -39,20 +42,18 @@
   "ui.options.multiplayer.showPlayerIDs": "Pokazuj identyfikatory graczy",
   "ui.options.multiplayer.showPlayerIDs.tooltip": "Włącz, aby widzieć identyfikatory graczy na liście",
   "ui.options.multiplayer.disableLuaReload": "Wyłącz ponowne ładowanie Lua przy opuszczaniu serwera wieloosobowego.",
-  "ui.options.multiplayer.disableLuaReload.tooltip": "Ponowne załadowanie Lua jest wymagane, aby zapewnić przywrócenie gry do stanu roboczego, jeśli modyfikacje zmieniły funkcje gry. Opcje można wyłączyć, aby zapewnić płynniejsze działanie, robisz to na własne ryzyko.",
+  "ui.options.multiplayer.disableLuaReload.tooltip": "Ponowne załadowanie Lua jest wymagane, aby zapewnić przywrócenie gry do stanu roboczego, w przypadku gdy modyfikacje zmieniły funkcje gry. Opcje można wyłączyć, aby zapewnić płynniejsze działanie, robisz to na własne ryzyko.",
   "ui.options.multiplayer.disableInstabilityPausing": "Wyłącz pauzę spowodowaną niestabilnością",
   "ui.options.multiplayer.disableInstabilityPausing.tooltip": "Dzięki temu rozgrywka będzie płynniejsza, ale może powodować problemy z synchronizacją po kilku niestabilnościach",
-  "ui.options.multiplayer.simplifyRemoteVehicles": "Używaj uproszczonych pojazdów, jeśli jest to możliwe",
-  "ui.options.multiplayer.simplifyRemoteVehicles.tooltip": "To ustawienie zastąpi zdalne pojazdy wersjami \"ulicznymi\" dostarczonymi przez BeamNG. Zmniejsza dokładność kolizji i efekty wizualne, kosztem poprawienia wydajność",
-  "ui.options.multiplayer.enableBlobs": "Włącz obiekty blob dla niezespawnowanych pojazdów",
+  "ui.options.multiplayer.simplifyRemoteVehicles": "Używaj uproszczonych modeli pojazdów, jeśli jest to możliwe",
+  "ui.options.multiplayer.simplifyRemoteVehicles.tooltip": "To ustawienie zastąpi zdalne pojazdy wersjami \"ulicznymi\" dostarczonymi przez BeamNG. Opcja ta zmniejsza dokładność kolizji i efekty wizualne, kosztem poprawienia wydajność",
+  "ui.options.multiplayer.enableBlobs": "Włącz bloby dla niezespawnowanych pojazdów",
   "ui.options.multiplayer.enableBlobs.tooltip": "Niespawnowane pojazdy będą widoczne jako kolorowe kule",
   "ui.options.multiplayer.showSpectators": "Pokaż  nazwe gracza z poziomu widza pod tabliczką z nazwą pojazdu",
   "ui.options.multiplayer.showSpectators.tooltip": "Gracze obserwujący pojazdy innych osób zostaną wyświetleni pod ich nazwami użytkownika",
   "ui.options.multiplayer.spectatorUnifiedColors": "Ten sam kolor nazw użytkowników dla obserwujących",
   "ui.options.multiplayer.spectatorUnifiedColors.tooltip": "Renderuj wszystkie nazwy oberwujących na szaro, niezależnie od roli użytkownika",
   "ui.options.multiplayer.nametagFade": "Zanikanie/zanikanie nazw użytkowników",
```

```suggestion
"ui.options.multiplayer.nametagFade": "Zanikanie/zanikanie nazw użytkowników",
"ui.options.multiplayer.nametagFadeIn": "Rozjaśnij",
"ui.options.multiplayer.nametagFadeOut": "Zniknij",
```
Please accept the suggestions, then we can close and merge the PR